config/src/config/mempool_config.rs

```diff
@@ -132,6 +144,7 @@ impl ConfigOptimizer for MempoolConfig {
                 mempool_config.shared_mempool_batch_size = 200;
                 modified_config = true;
             }
+            mempool_config.num_sender_buckets = 1;
```

Comment:
you should only override if not set manually in the config, i.e. check the pattern above:
```
            if local_mempool_config_yaml["num_sender_buckets"].is_null() {
                mempool_config.num_sender_buckets = 1;
                modified_config = true;
            }
```

Reply:
Done.